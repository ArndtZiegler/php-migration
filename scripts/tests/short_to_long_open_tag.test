<?
<? echo "Test" ?>
<?
    echo "Test";
?>
    <?
    <a href="https://example.com">example</a><?
<?=$var;?>
<?= $var; ?>
<?xml version="1.0" encoding="UTF-8"?>
<?PHP
<?php