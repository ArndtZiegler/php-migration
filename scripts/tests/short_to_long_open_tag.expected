<?php
<?php echo "Test" ?>
<?php
    echo "Test";
?>
    <?php
    <a href="https://example.com">example</a><?php
<?=$var;?>
<?= $var; ?>
<?xml version="1.0" encoding="UTF-8"?>
<?PHP
<?php